package.json

```diff
@@ -1,97 +1,120 @@
 {
-  "name": "cvs-svc-document-retrieval",
-  "version": "1.0.0",
-  "description": "Microservice that retrieves documents",
-  "main": "handler.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/dvsa/cvs-document-retrieval.git"
-  },
+  "name": "cvs-svc-template",
+  "version": "1.1.1",
+  "description": "",
+  "main": "src/index.ts",
   "engines": {
-    "node": "10.18.1",
-    "npm": "5.6.0"
+    "node": "12.x",
+    "npm": "6.x"
   },
-  "scripts": {
-    "start": "BRANCH=local SLS_DEBUG=* serverless offline start",
-    "start:test": "npm run tslint && SLS_DEBUG=* BRANCH=local node_modules/gulp/bin/gulp.js start-serverless",
-    "end:test": "node_modules/gulp/bin/gulp.js kill-serverless",
-    "debug": "export SLS_DEBUG=* && BRANCH=local node --inspect ./node_modules/serverless/bin/serverless offline start",
-    "build": "node_modules/typescript/bin/tsc --rootDir ./ --outDir .build --sourceMap false && npm run build:copy",
-    "build:copy": "find src -type f \\( -name \"*.yml\" -or -name \"*.json\" \\) | cpio -pdm .build && find tests -type f \\( -name \"*.yml\" -or -name \"*.json\" \\)  | cpio -pdm .build",
-    "test": "BRANCH=local SLS_DEBUG=* AWS_XRAY_CONTEXT_MISSING=LOG_ERROR jest --testMatch=\"**/*.unitTest.ts\" --coverage",
-    "test-i": "BRANCH=local jest --testMatch=\"**/*.intTest.ts\" --runInBand",
-    "dynamodb": "sls dynamodb start",
-    "prepush": "BRANCH=local npm test && npm run tslint",
-    "predeploy": "npm install && npm run prepush",
-    "tslint": "tslint src/**/*.ts tests/**/*.ts -q",
-    "sonar-scanner": "AWS_XRAY_CONTEXT_MISSING=LOG_ERROR npm run tslint && AWS_XRAY_CONTEXT_MISSING=LOG_ERROR npm run coverage && sonar-scanner",
-    "coverage": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.*Test.ts\" --runInBand --coverage",
-    "audit-dep": "npm audit --json | node_modules/.bin/audit-filter --nsp-config=.nsprc --audit=-",
-    "package": "mkdir ${ZIP_NAME} && cp package.json package-lock.json ${ZIP_NAME}/ && cp -r .build/src/* ${ZIP_NAME}/ && cd ${ZIP_NAME} && npm ci --production && rm package.json package-lock.json && zip -qr ../${ZIP_NAME}.zip .",
-    "security-checks": "npm audit --audit-level=critical && git secrets --register-aws && git secrets --scan && git log -p | scanrepo",
-    "tools-setup": "echo 'nothing to do'"
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm-run-all audit lint prettier",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+      "pre-push": "npm-run-all security-checks build test"
```

Comment:
do we want to run `security-checks` on pre-commit instead as here we are allowing secrets to possibly be committed to the history. This can still be undone with rebasing but it would be best to have that running before we commuit.